crates/matrix-sdk/src/room/timeline/event_item.rs

```diff
@@ -224,6 +201,15 @@ impl PartialEq<TransactionId> for TimelineKey {
     }
 }
 
+/// The display name and avatar URL of a room member.
+#[derive(Clone, Debug)]
+pub struct Profile {
```

Comment:
Since I'm guessing this is for display purposes, we probably want to store `display_name_ambiguous` in the struct. So we know to disambiguate it when displaying it.

Reply:
Yeah, I didn't know we actually have that info.

Reply:
It's calculated when the member is extracted from the store, so it's available in `RoomMember::name_ambiguous()`